package/doc/sphinx/source/documentation_pages/loading_files/amber.rst

```diff
@@ -0,0 +1,137 @@
+.. _loading_amber:
+
+###################################
+Loading Amber files with MDAnalysis
+###################################
+
+MDAnalysis can read PRMTOP files as topologies and
+NCDF and ascii coordinate and restart files as trajectories from
+`Amber MD`_ simulations.
+Typically using NCDF as the trajectory format will give the best performance
+as all other trajectory formats are ascii based.
+
+.. _Amber MD: http://ambermd.org
+
+.. _load_amber_top:
+
+Loading Amber PRMTOP files
+--------------------------
+
+MDAnalysis reads `TOP format`_ files with an extension of ``TOP``,
+``PRMTOP`` or ``PARM7``.
+The below table shows the relationship between Amber flags and the attribute
+created in MDAnalysis:
+The ``type_indices`` attribute is unique to Amber formats and is
+an *integer* representation of the atom type, rather than the
+typical string representation found throughout MDAnalysis.
+
+.. table:: Mapping of Amber flags to MDAnalysis attributes
+
+  +-----------------+----------------------+
+  | AMBER flag      | MDAnalysis attribute |
+  +=================+======================+
+  | ATOM_NAME       | names                |
+  +-----------------+----------------------+
+  | CHARGE          | charges              |
+  +-----------------+----------------------+
+  | ATOMIC_NUMBER   | elements             |
+  +-----------------+----------------------+
+  | MASS            | masses               |
+  +-----------------+----------------------+
+  | ATOM_TYPE_INDEX | type_indices         |
+  +-----------------+----------------------+
+  | AMBER_ATOM_TYPE | types                |
+  +-----------------+----------------------+
+  | RESIDUE_LABEL   | resnames             |
+  +-----------------+----------------------+
+  | RESIDUE_POINTER | residues             |
+  +-----------------+----------------------+
+
+.. note::
+
+   The Amber charge is converted to electron charges as used in
+   MDAnalysis and other packages. To get back Amber charges, multiply
+   by 18.2223.
```

Comment:
Perhaps point to `MDAnalysis.units` where we store the value (and should also have written down what all these units are).

Reply:
done